flixel/animation/FlxAnimationController.hx

```diff
@@ -9,6 +9,11 @@ using StringTools;
 
 class FlxAnimationController implements IFlxDestroyable
 {
+	/**
+	 * The speed across every animation that is played.
+	 */
+	public static var animationSpeed:Float = 1.0;
```

Comment:
`sprite.animation.animationSpeed` is redundant and, imo not very intuitive. I say use the name `timeScale` since this performs a similar function to [FlxG.timeScale](https://api.haxeflixel.com/flixel/FlxG.html#timeScale)

Reply:
also just realizing this is static, I assumed this would be an instance var, so you can change the rate of a specific sprite's animation, I see very little use for a global animation speed

Reply:
Ok

Reply:
> also just realizing this is static, I assumed this would be an instance var, so you can change the rate of a specific sprite's animation, I see very little use for a global animation speed

^